lib/rules/no-restricted-imports.js

```diff
@@ -74,6 +74,9 @@ const arrayOfStringsOrObjectPatterns = {
                         minItems: 1,
                         uniqueItems: true
                     },
+                    importNamePatterns: {
```

Comment:
```suggestion
                    importNamePattern: {
```

Since it's only one pattern.